src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,43 @@
+package codeu.controller;
+
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import java.time.Instant;
+import codeu.model.data.Conversation;
+import codeu.model.data.User;
+import code.model.data.Message;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+
+
+public class AdminServlet extends HttpServlet {
+    List<String> admin = Arrays.asList("anAdmin", "Admin1", "Admin2");
```

Comment:
I think the type of the variable admin should be changed from a List to a Set. That way checking if a user is an admin will be a lot quicker(O(1) vs O(n)).

I think something like this should compile:
``` java
Set<String> admin = new HashSet<>(Arrays.asList("anAdmin", "Admin1", "Admin2"));
```